src/main/java/org/alliancegenome/curation_api/jobs/executors/GeneExecutor.java

```diff
@@ -37,25 +34,31 @@ public void execLoad(BulkLoadFile bulkLoadFile, Boolean cleanUp) {
 		log.info("Running with dataProvider : " + dataProvider.name());
 
 		IngestDTO ingestDto = readIngestFile(bulkLoadFile, GeneDTO.class);
-		if (ingestDto == null) return;
-		
+		if (ingestDto == null) {
+			return;
+		}
+
 		List<GeneDTO> genes = ingestDto.getGeneIngestSet();
-		if (genes == null) genes = new ArrayList<>();
+		if (genes == null) {
+			genes = new ArrayList<>();
+		}
 
 		List<Long> geneIdsLoaded = new ArrayList<>();
 		List<Long> geneIdsBefore = new ArrayList<>();
 		if (cleanUp) {
 			geneIdsBefore.addAll(geneService.getIdsByDataProvider(dataProvider));
 			log.debug("runLoad: Before: total " + geneIdsBefore.size());
 		}
-		
+
 		bulkLoadFile.setRecordCount(genes.size() + bulkLoadFile.getRecordCount());
 		bulkLoadFileDAO.merge(bulkLoadFile);
 
 		BulkLoadFileHistory history = new BulkLoadFileHistory(genes.size());
 		createHistory(history, bulkLoadFile);
 		boolean success = runLoad(geneService, history, dataProvider, genes, geneIdsLoaded);
-		if(success && cleanUp) runCleanup(geneService, history, bulkLoadFile, geneIdsBefore, geneIdsLoaded);
+		if(success && cleanUp) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.